modules/op-starting-a-task-run-pipeline-run-build-user.adoc

```diff
@@ -0,0 +1,125 @@
++
+.Example: A task run that runs Buildah as user id `1000`
+[source,yaml]
+----
+apiVersion: v1
+data:
+  Dockerfile: |
+    ARG BASE_IMG=registry.access.redhat.com/ubi8/ubi
+    FROM $BASE_IMG AS buildah-runner
+    RUN dnf -y update && \
+        dnf -y install git && \
+        dnf clean all
+    CMD git
+kind: ConfigMap
+metadata:
+  name: dockerfile <1>
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: buildah-as-user-1000
+spec:
+  serviceAccountName: pipelines-sa-userid-1000
+  params:
+  - name: IMAGE
+    value: image-registry.openshift-image-registry.svc:5000/test/buildahuser
+  taskRef:
+    kind: Task
+    name: buildah-as-user
+  workspaces:
+  - configMap:
+      name: dockerfile <2>
+    name: source
+----
+<1> You can use a config map because the focus is on the task run, without any prior task that fetches some sources with a Dockerfile. 
+<2> You can mount a config map as the source workspace for the `buildah-as-user` task.
+
+. (Optional) You can create a pipeline and a corresponding pipeline run.
++
+.Example: A pipeline and corresponding pipeline run
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pipeline-buildah-as-user-1000
+spec:
+  params:
+  - name: IMAGE
+  - name: URL
+  workspaces:
+  - name: shared-workspace
+  - name: sslcertdir
+    optional: true
+  tasks:
+  - name: fetch-repository <1>
+    taskRef:
+      name: git-clone
+      kind: ClusterTask
+    workspaces:
+    - name: output
+      workspace: shared-workspace
+    params:
+    - name: url
+      value: $(params.URL)
+    - name: subdirectory
+      value: ""
+    - name: deleteExisting
+      value: "true"
+  - name: buildah
+    taskRef:
+      name: buildah-as-user <2>
+    runAfter:
+    - fetch-repository
+    workspaces:
+    - name: source
+      workspace: shared-workspace
+    - name: sslcertdir
+      workspace: sslcertdir
+    params:
+    - name: IMAGE
+      value: $(params.IMAGE)
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-buildah-as-user-1000
+spec:
+  serviceAccountName: pipelines-sa-userid-1000
+  params:
+  - name: URL
+    value: https://github.com/openshift/pipelines-vote-api
+  - name: IMAGE
+    value: image-registry.openshift-image-registry.svc:5000/test/buildahuser
+  taskRef:
+    kind: Pipeline
+    name: pipeline-buildah-as-user-1000
+  workspaces:
+  - name: shared-workspace <3>
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 100Mi
+----
+<1> Use the `git-clone` cluster task to fetch the source containing a Dockerfile and build it using the modified Buildah task.
+<2> Refer to the modified Buildah task.
+<3> To share data between the `git-clone` task and the modified Buildah task, use a persistent volume claim (PVC) created automatically by the controller.
```

Comment:
For consistency, I would have all three steps be a command instead of leading step 3 with an infinitive phrase. 

3. Share data between the `git-clone` task and the modified Buildah task by using a persistent volume claim (PVC) created automatically by the controller.